src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3525,8 +3525,8 @@ public override void Accept(OperationVisitor visitor)
     /// </summary>
     internal abstract partial class BaseReturnStatement : Operation, IReturnStatement
     {
-        protected BaseReturnStatement(bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
-                    base(OperationKind.ReturnStatement, isInvalid, syntax, type, constantValue)
+        protected BaseReturnStatement(OperationKind kind, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
```

Comment:
>OperationKind kind [](start = 38, length = 18)

It feels like we should throw or assert if Kind is not what we expect. We are making strong assumptions about the value in Accept implementation.